portfolio/src/main/webapp/statistics.html

```diff
@@ -105,9 +107,54 @@ <h2 class="section-heading text-uppercase">Stop Killing Us</h2>
           <h3 class="section-subheading text-muted">
             Find the locations of historically black sites, recent protests, and police brutality victims.
           </h3>
+          <div class="dropdown">
+            <button
+              type="button"
+              class="btn btn-primary dropdown-toggle"
+              data-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+            >
+              Where martyrs have died
+            </button>
+            <ul class="dropdown-menu">
+              <li><a onclick="codeAddress('Minneapolis, Minnesota')">George Floyd</a></li>
```

Comment:
I would suggest you use javascript to generate this list instead of hard code every single one.

Reply:
Added